hazelcast/include/hazelcast/client/exception/protocol_exceptions.h

```diff
@@ -351,6 +356,13 @@ class HAZELCAST_API query : public hazelcast_
     boost::uuids::uuid originating_member_uuid_;
 };
 
+class HAZELCAST_API invocation_might_contain_compact_data : public hazelcast_
+{
+public:
+    explicit invocation_might_contain_compact_data(
+      const spi::impl::ClientInvocation& invocation);
```

Comment:
```suggestion
      const std::string &source, const spi::impl::ClientInvocation& invocation);
```
Should we add the source of the exception?

Reply:
Fixed.